pycqed/measurement/waveform_control/sequence.py

```diff
@@ -17,6 +20,8 @@ class Sequence:
     AWGs sequentially.
     """
 
+    RENAMING_SEPARATOR = "+"
```

Comment:
I know that this does not belong here, but into a different pull request. But it was part of a timer commit and reverting it now and reintroducing it later has the potential to cause more trouble than just leaving it here already.